test/cpp/test_tensor.cpp

```diff
@@ -1,18 +1,75 @@
 #include <gtest/gtest.h>
 
+#include <limits>
 #include <vector>
 
 #include <ATen/ATen.h>
 #include "cpp_test_util.h"
-#include "tensor.h"
 #include "torch/csrc/autograd/variable.h"
+#include "torch_xla/csrc/tensor.h"
+#include "torch_xla/csrc/tensor_util.h"
 #include "torch_xla_test.h"
 
 namespace torch_xla {
 namespace cpp_test {
+namespace {
+
+bool IsBidirectionalConversion(
```

Comment:
This name is confusing, I'd rather have a name which reflects that it's a test helper. How about `CheckBidirectionalConversion`?

Reply:
SGTM